webapp-django/crashstats/crashstats/migrations/0001_initial.py

```diff
@@ -1,6 +1,5 @@
 # -*- coding: utf-8 -*-
 # Generated by Django 1.11.15 on 2018-09-25 15:40
-from __future__ import unicode_literals
```

Comment:
This affects what happens to unicode when printing to a terminal that doesn't specify its encoding. This import would cause Python 2 to do what Python 3 does. Python 3 does this by default, so we don't need these anymore.

Having said that, I think this is part of the Django template for migration files, so it's possible more will pop up over time.